nedap/formatting-stack
# Use refactor-nrepl 3.3.1

## Brief

https://github.com/clojure-emacs/refactor-nrepl/blob/bc7779fc0d25ec025486438ad0f5b0990644a742/CHANGELOG.md#330

Now refactor-nrepl honors .clj-kondo config, as far as ns cleaning goes.

Which for us is useful: it means that f-s also gets to reuse kondo config, which means a DRYer/easier experience for a variety of users.

## QA plan

* Add sample :unused-namespace kondo config ([example](https://github.com/clojure-emacs/refactor-nrepl/blob/bc7779fc0d25ec025486438ad0f5b0990644a742/.clj-kondo/config.edn#L45-L46))
* Verify that f-s will honor that setting

## Author checklist



* Additionally, I have code-reviewed iteratively the PR considering the following aspects in isolation:

## Reviewer checklist

* Additionally, I have code-reviewed iteratively the PR considering the following aspects in isolation:
